randovania/gui/ui_files/preset_am2r_patches.ui

```diff
@@ -92,7 +92,7 @@
            <item>
             <widget class="QLabel" name="septogg_helpers_label">
              <property name="text">
-              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Septoggs will appear in certain rooms, helping you reach higher platforms if you don't have the means to reach them yourself. Due to SR-388's cave structure, this setting is &lt;b&gt;highly recommended&lt;/b&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Septoggs will appear in certain rooms, helping you reach higher platforms if you don't have the means to reach them yourself. Due to SR-388's cave structure, this setting is highly&lt;span style=&quot; font-weight:600;&quot;/&gt;recommended.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Just noticed there's a weird font-weight that should probably not be here, I'll remove it later